src/Sarif/Readers/DeferredList.cs

```diff
@@ -40,12 +40,16 @@ public class DeferredList<T> : IList<T>, IDisposable
 
         private Func<T, T> _transformer;
 
+        private T _lastItem;
```

Comment:
>_last [](start = 18, length = 5)

My only comment is about naming: Whenever I see "last" I immediately wonder "does this mean 'last in order' or 'most recent'?". In this case, it turned out to mean "most recent", but I had to carry that question in my mind from the time I read this line until I understood the answer from the setter below.

Consider `_mostRecentItem` or `_lastAccessedItem`.

Reply:
Reasonable. Fixed.